src/app/forms/form-common-options/form-common-options.js

```diff
@@ -0,0 +1,192 @@
+//     { label: "Beaver", value: "Beaver" },
+//     { label: "Box Elder", value: "Box Elder" },
+//     { label: "Cache", value: "Cache" },
+//     { label: "Carbon", value: "Carbon" },
+//     { label: "Daggett", value: "Daggett" },
+//     { label: "Davis", value: "Davis" },
+//     { label: "Duchesne", value: "Duchesne" },
+//     { label: "Emery", value: "Emery" },
+//     { label: "Garfield", value: "Garfield" },
+//     { label: "Grand", value: "Grand" },
+//     { label: "Iron", value: "Iron" },
+//     { label: "Juab", value: "Juab" },
+//     { label: "Kane", value: "Kane" },
+//     { label: "Millard", value: "Millard" },
+//     { label: "Morgan", value: "Morgan" },
+//     { label: "Piute", value: "Piute" },
+//     { label: "Rich", value: "Rich" },
+//     { label: "Salt Lake", value: "Salt Lake" },
+//     { label: "San Juan", value: "San Juan" },
+//     { label: "Sanpete", value: "Sanpete" },
+//     { label: "Sevier", value: "Sevier" },
+//     { label: "Summit", value: "Summit" },
+//     { label: "Tooele", value: "Tooele" },
+//     { label: "Uintah", value: "Uintah" },
+//     { label: "Utah", value: "Utah" },
+//     { label: "Wasatch", value: "Wasatch" },
+//     { label: "Washington", value: "Washington" },
+//     { label: "Wayne", value: "Wayne" },
+//     { label: "Weber", value: "Weber" }
+// ];
+
+// export const courtTypeOptions = [
+//     { label: "District Court", value: "District" },
+//     { label: "Justice Court", value: "Justice" }
+// ];
+
+// export function judicial(value) {
+//     var JudicialDistrict = "";
+//     switch (value) {
+//         case "Box Elder":
+//         case "Rich":
+//         case "Cache":
+//             JudicialDistrict = "First";
+//             break;
+//         case "Davis":
+//         case "Morgan":
+//         case "Weber":
+//             JudicialDistrict = "Second";
+//             break;
+//         case "Salt Lake":
+//         case "Summit":
+//         case "Tooele":
+//             JudicialDistrict = "Third";
+//             break;
+//         case "Juab":
+//         case "Millard":
+//         case "Utah":
+//         case "Wasatch":
+//             JudicialDistrict = "Fourth";
+//             break;
+//         case "Beaver":
+//         case "Iron":
+//         case "Washington":
+//             JudicialDistrict = "Fifth";
+//             break;
+//         case "Garfield":
+//         case "Kane":
+//         case "Piute":
+//         case "Sanpete":
+//         case "Sevier":
+//         case "Wayne":
+//             JudicialDistrict = "Sixth";
+//             break;
+//         case "Carbon":
+//         case "Emery":
+//         case "Grand":
+//         case "San Juan":
+//             JudicialDistrict = "Seventh";
+//             break;
+//         case "Daggett":
+//         case "Duchesne":
+//         case "Uintah":
+//             JudicialDistrict = "Eighth";
+//             break;
+//     }
+//     return JudicialDistrict;
+// }
```

Comment:
do you want to leave this commented code in this PR?

Reply:
Yes the reason I left that commented code in the PR is I wasn't sure what the best practice for something like this would be.

Reply:
gotcha. I think either can be good. personally, I like the individual exports better. 

then when you import you can do something like: 
```
 import { courtTypeOptions, countyOptions } from '/wherever'
```
There's one less level to dig into. Doing it this way can also be better for bundles splitting

Reply:
Yeah individual exports are more common - let's go with those.